clusterloader2/pkg/measurement/util/runtimeobjects/runtimeobjects.go

```diff
@@ -316,7 +315,7 @@ func GetNumSchedulableNodesMatchingSelector(c clientset.Interface, nodeSelector
 	}
 	var numSchedulableNodes int32
 	for _, node := range list {
-		if util.IsNodeSchedulableAndUntainted(&node) {
+		if !node.Spec.Unschedulable {
```

Comment:
Please make this a private function so that we won't start using it from other places.

Reply:
Done.